### DISCUSSION TITLE
我有堆字体文件，在主进程和渲染进程都会用到，需要在主进程和渲染进程个弄一份吗？

### DISCUSSION THREAD

**Discussion by lqomg** [alex8088/electron-vite#267] [Q&A] [answered]
我有堆字体文件，在主进程和渲染进程都会用到，需要在主进程和渲染进程个弄一份吗？还是有什么替代方案呢

**Comment by alex8088** [comment, marked as answer]
可以将字体资源放在resources中，渲染进程通过file:///协议或者自定义协议加载

**Reply by lqomg** [reply]
css中注册字体的时候可以使用file://协议吗？

**Reply by lqomg** [reply]
试了一下是可以的 只是要把路径中的“\“ replace成 “//”     谢谢大佬！ ！
